### ISSUE TITLE
Pikaday is not accessible while running a screenreader when attached to button

### ISSUE THREAD

**Issue by KJNC** [Pikaday/Pikaday#859]
Hi,

Pikaday is not accessible when triggered by a button and you have a screenreader (NVDA as example) running.

You cannot use the keyboard navigating the datepicker when in this situation.
